src/azure-cli/azure/cli/command_modules/netappfiles/custom.py

```diff
@@ -0,0 +1,154 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+
+from knack.log import get_logger
+from azure.mgmt.netapp.models import ActiveDirectory, NetAppAccount, NetAppAccountPatch, CapacityPool, CapacityPoolPatch, Volume, VolumePatch, VolumePropertiesExportPolicy, ExportPolicyRule, Snapshot
+from azure.cli.core.commands.client_factory import get_subscription_id
+
+logger = get_logger(__name__)
+
+# RP expted bytes but CLI allows integer TiBs for ease of use
+gib_scale = 1024 * 1024 * 1024
+tib_scale = gib_scale * 1024
+
+
+def str2bool(v):
+    return v.lower() in ("yes", "true", "t", "1")
```

Comment:
It would be more typical and straightforward to convert these "string-based flags" to actual flags. If these are behavior flags for the command, registering the arguments with `action='store_true'` would do this. If these are boolean properties on a resource, then using the CLI's built-in `get_three_state_flag()` would be the preferred mechanism.

Reply:
Ok, will update to use get_three_state_flag()

Reply:
This is updated in the current version